app_dart/lib/src/model/ci_yaml/ci_yaml.dart

```diff
@@ -97,4 +119,61 @@ class CiYaml {
 
     return regexp.hasMatch(branch);
   }
+
+  void _validateSchedulerConfig(pb.SchedulerConfig schedulerConfig, {pb.SchedulerConfig? totConfig}) {
```

Comment:
Can you add a dart doc here with an overview of the validations?

Reply:
A dart doc is a comment in the code

See https://dart.dev/guides/language/effective-dart/documentation

Reply:
Sorry for the misunderstanding, I have removed the google doc link and added in a real dart doc